### DISCUSSION TITLE
Any roadmap? features of the v1? when do you plan to deliver the v1?

### DISCUSSION THREAD

**Discussion by Abdelkrim** [docmost/docmost#569] [General]
Hi,

Do you know where I could find a roadmap?

What kind of features will the v1 support?

When do you plan to deliver the v1? 

Thank you,

**Comment by Shousokon** [comment]
I'd also like to know if there is a public roadmap since I wasn't able to find any

**Comment by arstagaev** [comment]
+1 agree, still wait it :-)
